x-pack/platform/plugins/shared/alerting_v2/kibana.jsonc

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "plugin",
+  "id": "@kbn/alerting-v2-plugin",
+  "owner": ["@elastic/response-ops"],
+  "group": "platform",
+  "visibility": "shared",
+  "plugin": {
+    "id": "alerting_esql",
+    "browser": false,
+    "server": true,
+    "configPath": ["xpack", "alerting_v2"],
```

Comment:
This should change before merging to main. We need to find a way to have `xpack.alerting` while being on another plugin. Maybe it is permitted 🙂.